.github/workflows/frontend-unit-tests.yml

```diff
@@ -17,6 +13,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install modules
-        run: yarn
-      - name: Run tests
+        run: yarn --frozen-lockfile
+      - name: Run codegen
+        run: yarn codegen
```

Comment:
We need to run `codegen` to run unit tests because they rely on auto-generated imports.